app/models/form_profiles/va10203dny.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'evss/gi_bill_status/service'
+
+module VA10203dny
+  FORM_ID = '22-10203DNY'
+
+  class FormInstitutionInfo
+    include Virtus.model
+
+    attribute :name, String
+    attribute :city, String
+    attribute :state, String
+    attribute :country, String
+  end
+
+  class FormEntitlementInformation
+    include Virtus.model
+
+    attribute :months, Integer
+    attribute :days, Integer
+  end
```

Comment:
This code is duplicate of VA10203. Can we make it common to both?

Reply:
I don't think we need this because the form_profile is used for the form prefill, but we won't have anyone actually filling out a va10203dny from the website.

Reply:
Good point. Makes sense.